chore(rpa): add first documentation for RPA bridge (#643)
camunda/camunda-docs-manual: content/installation/camunda-bpm-rpa-bridge.md

---

title: "Get Started with the Camunda RPA Bridge"
weight: 50

menu:
  main:
    name: "Camunda BPM RPA Bridge"
    identifier: "installation-camunda-bpm-rpa-bridge"
    parent: "installation-guide"
    pre: ""
---

This page describes the steps to orchestrate and execute RPA bots from Camunda BPM using the Camunda BPM RPA Bridge.

{{< enterprise >}}

>   Please note that the RPA bridge is only available as enterprise edition.

Reviewer: I think this should state that the bridge is available "in" or "through" the EE version. Or for EE customers.
```suggestion
  Please note that the RPA bridge is only available in the Enterprise Edition.
```

Author: I don't think this change is necessary as the bridge will not be bundled within something else (_in_ the Enterprise Edition). It is an Enterprise Edition itself and there is no Community Edition.

What do you think about this?
```suggestion
  Please note that the RPA bridge is only available for customers with access to our enterprise artifacts.
```

Reviewer: Looks good! :+1:

{{< /enterprise >}}

# Requirements

## Java
Please make sure that you have the Java Runtime Environment 8+ installed.

You can verify this by using your terminal, shell, or command line:

```sh
java -version
```

> If you need to install Java Runtime Environment, you can [find the download from Oracle here](https://www.oracle.com/java/technologies/javase-downloads.html).

Reviewer: ```suggestion
If you need to install a Java Runtime Environment, you can [find the download from Oracle here](https://www.oracle.com/java/technologies/javase-downloads.html).
```

## Camunda BPM
You will need a running Camunda BPM Enterprise Edition with Camunda 7.14 or later. You can find all downloadable distros [here](https://downloads.camunda.cloud/enterprise-release/camunda-bpm/).

## Cawemo and Camunda Modeler
The easiest way to create a BPMN model that connects to an RPA bot is by using Cawemo to create a worker catalog that you can apply to your process model using the Camunda Modeler. Make sure to use Cawemo 1.4 (or later) and Camunda Modeler 4.2 (or later).

# Installation Procedure

> 1. Downlaod the Camunda RPA Bridge distro from [here](). TODO

Reviewer: Typo. Also the download link needs to be added.
```suggestion
1. Download the Camunda RPA Bridge distro from [here](). TODO
```

1. Extract the downloaded archive

> 1. Configure the bridge with the shipped `application.yml` file (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md" >}}) for a full list of configuration options).

Reviewer: We can further specify the section of the User Guide docs:
```suggestion
1. Configure the bridge with the shipped `application.yml` file (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#camunda-bpm-rpa-bridge-configuration" >}}) for a full list of configuration options).
```

1. Start the Bridge by executing the jar file: `java -jar camunda-bpm-rpa-bridge.jar`

> 1. create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md" >}} for more information)

Reviewer: Fix typo. We can also further specify the section of the docs.
```suggestion
1. Create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#set-up-an-rpa-task" >}} for more information)
```

> 1. Create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#set-up-an-rpa-task" >}} for more information)

Reviewer 2: ```suggestion
1. Create and deploy a model that includes an RPA service task (see the [user guide]({{< ref "/user-guide/camunda-bpm-rpa-bridge.md#set-up-an-rpa-task" >}}) for more information)
```
